alcs-frontend/src/app/features/application/lfng-info/lfng-info.component.html

```diff
@@ -121,18 +121,24 @@ <h4>Plans & Bylaws: Zoning</h4>
           <div>
             <div class="subheading2">Zoning Bylaw Name</div>
             <div>{{ applicationReview.zoningBylawName }}</div>
+            <app-no-data *ngIf="!applicationReview.zoningBylawName"></app-no-data>
           </div>
           <div>
             <div class="subheading2">Zoning Designation</div>
             <div>{{ applicationReview.zoningDesignation }}</div>
+            <app-no-data *ngIf="!applicationReview.zoningDesignation"></app-no-data>
           </div>
           <div>
             <div class="subheading2">Minimum Lot Size (hectares)</div>
             <div>{{ applicationReview.zoningMinimumLotSize }}</div>
+            <app-no-data *ngIf="!applicationReview.zoningMinimumLotSize"></app-no-data>
           </div>
           <div>
             <div class="subheading2">Is this proposal consistent with the current zoning designation?</div>
-            <div>{{ applicationReview.isZoningConsistent ? 'Yes' : 'No' }}</div>
+            <div *ngIf="applicationReview.OCPConsistent !== null">
```

Comment:
Should this be isZoningConsistent not OCPConsistent?

Reply:
Yes, you're right. Thanks for the catch!

Reply:
Fixed!